test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -153,14 +154,14 @@ public void HtmlAttributes_IgnoresCase(string originalName, string updatedName)
         {
             // Arrange
             var executionContext = new TagHelperExecutionContext("p", selfClosing: false);
-            executionContext.HTMLAttributes[originalName] = "hello";
+            executionContext.HTMLAttributes[originalName].Value = "hello";
```

Comment:
correction: can't have the `TagHelperAttribute.Name` throw if it's `null` because we need to inspect it in order to set it in our string setter. That's fine though. Will jsut throw when consuming it.